app/models/solr_document.rb

```diff
@@ -19,7 +19,7 @@ class SolrDocument
   use_extension(LinkedData)
 
   def resource
-    @resource ||= Valkyrie::MetadataAdapter.find(:index_solr).resource_factory.to_resource(object: to_h)
+    @resource ||= Valkyrie::MetadataAdapter.find(:indexing_persister).query_service.find_by(id: id)
```

Comment:
is it just this line that converts the catalog show page to use a record out of
Postgres instead of Solr? I'd rather see that change in its own commit (fine with me to be in this PR).